### ISSUE TITLE
A11y_Windows Admin Center (WAC) - Full Test Pass_Networks_Screen Reader: NVDA is not reading selected row in the grid  as selected

### ISSUE THREAD

**Issue by Shanmukha39** [nvaccess/nvda#10250]
### Steps to reproduce:
login with the appropriate credentials
2.Tab till server "wacpartner.redmond.corp.microsoft.com" and press enter
3.Tab till "Local Users & Groups" in tools in the left navigation bar and press enter
4.Tab till "Groups" and select any of the groups in the grid so that details section will open
5.Start NVDA
6.Verify whether NVDA is reading selected row present in the grid as selected

### Actual behavior:
NVDA is not reading selected row in the grid  as selected
### Expected behavior:
NVDA should read selected row as selected

**Comment by Adriani90** [comment]
@Shanmukha39 in which browser did you see this behavior? Is it also reproducible with NVDA 2019.2?

**Comment by Shanmukha39** [comment]
@Adriani90 
I have checked in Chrome browser, it is reproducible with NVDA 2019.2 as well

**Comment by Adriani90** [comment]
@Shanmukha39 this issue cannot be reproduced by everyone due to missing credentials, so no way to investigate here. Are you still having this issue with NVDA 2020.1 in Chrome 83?
Do you have a code snippet or a minimal test case for us to reproduce? Otherwise please reach out directly to NV Access and provide them with credentials so that they can help in solving this issue directly.
I am closing this issue for now but please comment with a minimal test case annd we can reopen afterwards. Thanks.
